### ISSUE TITLE
English Translation

### ISSUE THREAD

**Issue by Evo-c** [MicroCBer/orthrus#2]
Hi any chance of possibly adding English translation? I'm also happy to help provide translations if it's helpful at all.

嗨，请原谅我的语法错误，我正在使用在线翻译器。
能不能加上英文翻译？ 如果您愿意，我很乐意帮助您做到这一点。

**Comment by std-microblock** [comment]
actually you can use English to talk to me directly if you want xD, I can understand it in most the situations

the problem with adding English translation is I didn't expect it to be used world-widely, so there would be some work to add i18n to it. Translating it to English won't be a big problem I assume.

Anyway, I'll make an English-translated version when I have spare time, also if you can translate the texts and send them to me, I would appreciate that!

**Comment by std-microblock** [comment]
There are only a few texts that should be translated.

```
            { "[物品] 解锁所有物品", "items.add-unlockables-stashpoint"},
            {"[物品] 物品数量*100","items.add-unlockables-num" },
            {"[物品] “所有物品”物品","items.all-items" },
            {"[物品] 解锁所有西装","items.add-unlockables-disguise" },
            {"[入口] 解锁所有入口","entrance.unlock-all" },
            {"[计划] 解除计划限制（西装等）","planning.remove-limits" },
            {"[计划] 解锁全部物品藏匿点","agency-pickups.unlock-all" },
            {"[物品] 添加地图内的伪装到西装", "items.add-map-disguises"},
            {"[物品] 额外物品包 1","items.extra-items-1" },
            {"[物品] 自定义 RepoID (放入 custom-items.json )","items.custom-repoid" },
            {"[游戏] 去除失败惩罚（自由人）","gameplay.no-fail-punishment" },
            {"[自由人] 自带所有物品","freelancer.all-items" }
```

( the id on the right side is already can be considered as an English version of the description lol )

**Comment by std-microblock** [comment]
oh I see you've done the translation, I'll use it directly when I 'm free, greatly thanks!

**Comment by Evo-c** [comment]
> oh I see you've done the translation, I'll use it directly when I 'm free, greatly thanks!

Yes sorry, I was going to add a language box for selectable strings then submit a pull request but in the end I figured it's probably best to wait to hear back from you! Also very clever how you've intercepted the requests and even made the program patch the certificate pinning, it works great!

I've been playing around with the game the past few days and found it's actually possible to launch the time-limited elusive contracts while online and still receive all the unlocks from completing them by just modifying the header and responses, do you think this would be suitable to add? If so I'm happy to discuss and help with it, let me know if so also if you have Discord or something I can discuss if you like?🙂

**Comment by std-microblock** [comment]
Discord: MicroBlock#4243
Telegram: https://t.me/microblock0

I don't use these two software so frequently, so it might be normal if I cannot reply to you in time, we usually use QQ in China.

Thanks for your work! :)

**Comment by std-microblock** [comment]
( Actually, I've done a tool to get unlockables of ETs, whose name is [hitman3-ac](https://microblock.cc/hitman3-ac)(hitman3 auto-complete), but due to it has a really high possibility to be used as a cheating tool as it can submit a completely fake score, I neither open-sourced it nor publicized it. Recently I've enhanced it to enable it to make one reach Lv.100 in Freelancer in ~5mins, and there would be more functions for it. I strongly suggest you not tell this bug to anyone except ioi, as it has the ability to destroy the ranks )
